### PR TITLE
place AggregationDistinct on oss

### PR DESCRIPTION
**Pull request by timber-lin**

### CODE REVIEW

File: velox/core/PlanNode.h [hunk 1]
```diff
@@ -524,6 +524,7 @@ class AggregationNode : public PlanNode {
       const std::vector<FieldAccessTypedExprPtr>& preGroupedKeys,
       const std::vector<std::string>& aggregateNames,
       const std::vector<Aggregate>& aggregates,
+      const std::vector<bool>& aggregateDistincts,
```
Comment (mbasmanova) [inline velox/core/PlanNode.h:527]: Is this a duplicate of Aggregate::distinct ?

**Comment by mbasmanova** [comment]
See https://github.com/facebookincubator/velox/pull/5632

**Comment by shiyu-bytedance** [comment]
> See #5632

@mbasmanova yes

> See #5632

hi @mbasmanova yes, this change implements the same support as #5632. However, our implementation also supports distinct multiple-channel input aggregates, such as covar_pop().

**Comment by mbasmanova** [comment]
@shiyu-bytedance 

> hi @mbasmanova yes, this change implements the same support as https://github.com/facebookincubator/velox/pull/5632. Though this implementation also supports distinct multiple-channel input aggregates, such as covar_pop(x,y), covar_samp(x,y)

Would you open a GitHub issue to describe missing functionality and the way you propose to add it? It would be nice to see if the solution in #5632 can be extended.

**Comment by timber-lin** [comment]
this PR is no longer needed

**Pull request closed without merging**